pkgs/by-name/do/dockrtui/package.nix

```
@@ -0,0 +1,31 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  nix-update-script,
+}:
+
+rustPlatform.buildRustPackage rec {
```

Lets use finalAttrs nixpkgs pattern instead of rec to avoid its drawbacks!
- #315337
```suggestion
rustPlatform.buildRustPackage (finalAttrs: {
```